### ISSUE TITLE
Display SMTP message when sending

### ISSUE THREAD

**Issue by marcvs** [neomutt/neomutt#982]
## bug reports

* Expected behavior
When in a network that does not allow smtp, our network guys display an informational message why one cannot send email. My older mutt did display this message

* Actual behavior
No message dsplayed, message not sent. I'm left in the same dialogue without acutally sending the mail.

* Steps to reproduce
Find a network that returns an erronous message when attemping outgoing 25 connections

* Used program versions
NeoMutt 20171027-148-33b649

* Operating System and its version
linux 4.9.0-2-amd64 (debian/testing)

**Comment by flatcap** [comment]
Thanks for the bug report, @marcvs 

> My older mutt did display this message

Hmm...  Occasionally, when a message gets lost it's actually been overwritten (or cleared) before the user can read it.
Try running with logging enabled `neomutt -d2` and see if the message shows up in `~/.neomuttdebug0`

**Comment by marcvs** [comment]
Yes it shows up in that log.

In old mutt this message is displayed in the last line on screen.

**Comment by flatcap** [comment]
> Yes it shows up in that log.

OK, we're getting the message and it's being displayed.
After an error, neomutt goes back to the compose screen and waits for input.

I've tested the scenario and I see the messages every time.
I've tested unknown users and had the server return meaningless error codes.
I've also tried tmux and screen (just in case).
I've tried resizing the screen to see if that clears the message.  It doesn't.

Some of the other SMTP code sleeps after an error message, so I've added one in branch [[issue/982](https://github.com/neomutt/neomutt/tree/issue/982)]
The length of the sleep will depend on the `$sleep_time` variable.
